data/transactions/logic/evalStateful_test.go

```diff
@@ -467,6 +470,21 @@ func (l *testLedger) GetDelta(txn *transactions.Transaction) (evalDelta basics.E
 	return
```

Comment:
I believe you should also add the logs the return value of the `GetDelta` function.